build.sbt

```diff
@@ -3,3 +3,5 @@ scalaVersion in ThisBuild := "2.12.4"
 crossScalaVersions in ThisBuild := Seq("2.11.11", "2.12.4")
 
 organization in ThisBuild := "org.coursera.naptime"
+
+scalafmtOnCompile in ThisBuild := true
```

Comment:
even if we choose to use default formatting, i think itd also be a good idea to keep a copy of scalafmt.conf so we can at a glance see what is being applied (and also able to keep the settings independent of version bumps)

Reply:
Good idea. Let me do that and point to `scalafmt.conf`. We can copy the one from `infra-services` to start.